### PR TITLE
Add breadcrumbs component

### PR DESCRIPTION
**Pull request by StianHaugland1**
image

### CODE REVIEW

**Review by kentis — APPROVED** [review]

File: h5p-bildetema/package.json [hunk 1]
```diff
@@ -28,9 +28,11 @@
   },
   "dependencies": {
     "@mdx-js/react": "^2.1.1",
+    "@testing-library/react": "^13.3.0",
```
Comment (boyum) [inline h5p-bildetema/package.json:31]: Move to dev dependencies

File: h5p-bildetema/package.json [hunk 2]
```diff
@@ -28,9 +28,11 @@
...
     "h5p-utils": "^2.0.6",
     "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "react-dom": "^18.0.0",
+    "react-router-dom": "^6.3.0"
```
Comment (boyum) [inline h5p-bildetema/package.json:35]: Move to dev dependencies

**Comment by boyum** [comment]
The design uses a chevron instead of `>`. This is important because `>` might impose some value for screen readers (e.g. "larger than"), and they don't look alike, especially in some fonts. I suggest adding the chevron as an inline SVG.

```svg
<svg width="10" height="17" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M1.5 17 .067 15.567l6.6-6.6-6.6-6.6L1.5.933l8.034 8.034L1.5 17Z" fill="currentcolor"/></svg>
```

File: h5p-bildetema/src/components/Breadcrumbs/Breadcrumbs.test.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  const template = ({ breadcrumbs }: BreadcrumbsProps): HTMLElement => {
+    const { container } = render(
+      <MemoryRouter>
+        <Breadcrumbs breadcrumbs={breadcrumbs} />,
+      </MemoryRouter>,
+    );
+    return container;
+  };
+
+  it("No breadcrumbs should render nothing", () => {
```
Comment (boyum) [inline h5p-bildetema/src/components/Breadcrumbs/Breadcrumbs.test.tsx:17]: Test names should make sense to read after "is", e.g:

```suggestion
  it("should render no links if the list of breadcrumbs is empty", () => {
```

**Comment by boyum** [comment]
The name of the current page should be rendered in bold text.

File: h5p-bildetema/src/components/Breadcrumbs/Breadcrumbs.test.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    const container = template({
+      breadcrumbs: [{ label: "Tema", path: "/tema" }],
+    });
+    expect(container.textContent).toEqual("Tema,");
+    expect(container.querySelector("span")).toBeTruthy();
+    expect(container.querySelector("a")).toBeFalsy();
+    expect(container.querySelector("svg")).toBeFalsy();
+  });
+
+  it("Multiple breadcrumbs should render links", () => {
```
Comment (boyum) [inline h5p-bildetema/src/components/Breadcrumbs/Breadcrumbs.test.tsx:36]:
```suggestion
  it("should render breadcrumbs if there are multiple links", () => {
```

**Pull request merged**